cmd/mattermost/commands/server.go

```diff
@@ -82,6 +82,14 @@ func runServer(configStore config.Store, disableConfigWatch bool, usedPlatform b
 		return serverErr
 	}
 
+	if *configStore.Get().ServiceSettings.EnableLocalMode {
```

Comment:
Shouldn't it be part of the Server initialization and shutdown?

Reply:
Good point, moved there. 👍